test/acceptance-test.js

```diff
@@ -135,9 +178,27 @@ describe('public api', function() {
     });
   });
 
-  describe('Linter.prototype.verify', function() {
-    let basePath = path.join(fixturePath, 'with-errors');
+  describe('Linter.prototype.verify', async function() {
+    project = await createTempDir();
```

Comment:
Can you move this setup into a `before` (instead of in the root of the `describe`), and make the `describe` block back into a non-async function?